doc/modules/mixture.rst

```diff
@@ -183,8 +195,9 @@ similar to the one defined by :class:`GaussianMixture`.
 
 .. _variational_inference:
 
-Estimation algorithm: variational inference
----------------------------------------------
+|details-start|
+**Estimation algorithm: variational inference**
+|details-split|
```

Comment:
I would prefer not to hide this section, as it describes the pragmatics of the maths without going deeply into equations.

Reply:
Thanks for the review. Will revert the change made